### PR TITLE
Feat/chrome

### PR DESCRIPTION
**Pull request by coolguy1990**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      }
+    ],
+  ],
+  "presets": [
+    [
+      "env",
+      {
+        "targets": {
+          "browsers": [
+            "last 2 Chrome versions"
```
Comment (john-e) [inline .babelrc:22]: no love for IE ? 😆

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+10
```
Comment (john-e) [inline .nvmrc:1]: ❤️

File: manifest.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+  "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAq/S9dl/VwNTcp+T5ZLnOH8gOZoMkFTiKNsR6EDv3zUiEImaheCDMzFUhRiyxg1NY55RjB7oMcqFl2RQnMr8YIR4zmCN0VKljgzN4G81mH+tN+lSccKYOPcUed26nzzzSdxryUSg2QfadChtnCfFMaZxkwrQTn4So5pTyr9upPNDvL4BCXG1sxeZHpp5TDIJTNYlKngQzmWGd/yi0QBRwukeobk7nqiXzAoXkG2qHP3A+Y3YrGFsXfB3X1rM/1g8qnUHwYlv/MmLboOmRQdEBsKdstUmmGTQb6FuTAYVkTcaw/wtRHyd+Cdf9jTo2SVdU160zZ3Ngqr7B+cYvEx+htQIDAQAB",
```
Comment (john-e) [inline manifest.json:2]: what is this key ?

File: src/browsers/index.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+export interface Browser {
+  getBookmarkTree(): [];
+  parseBookmarks: any;
+  filterBookmarks: any;
+}
+
+let browser: Browser = {} as any;
+
+if (/Chrome/.test(navigator.userAgent)) {
+  browser = Object.assign(browser, chrome);
```
Comment (john-e) [inline src/browsers/index.ts]: DRY this.

File: src/components/ConsoleLog.tsx [hunk 5]
```diff
@@ -0,0 +1,7 @@
+const ConsoleLog = ({ children }: any): any => {
```
Comment (john-e) [inline src/components/ConsoleLog.tsx:1]: rename to `Logger` ?

**Review by john-e — APPROVED** [review]

File: .babelrc (hunk 1 above)
Comment (coolguy1990) [reply .babelrc:22]: not at the beginning. targeting chrome, then firefox. With the new Edge using chromium as base, shouldn't theoretically it work? not sure how their extensions works.

File: manifest.json (hunk 3 above)
Comment (coolguy1990) [reply manifest.json:2]: Its a catch 22 situation here. To load the extension for even developing/testing you need a key. But you can't get a key unless you release the extension. This key is just a random one for now. When we release, will add the proper one.

File: src/components/ConsoleLog.tsx (hunk 5 above)
Comment (coolguy1990) [reply src/components/ConsoleLog.tsx:1]: Thought of it. But planning to add a proper logging mechanism. This is just a temp solution for now, just to get this started.

File: src/browsers/index.ts (hunk 4 above)
Comment (coolguy1990) [reply src/browsers/index.ts]: ternary?

File: .babelrc (hunk 1 above)
Comment (john-e) [reply .babelrc:22]: JK

File: src/browsers/index.ts (hunk 4 above)
Comment (john-e) [reply src/browsers/index.ts]: Could do this.

```typescript
const browsers = {
   chrome,
   firefox
};

let browser = null;
Object.keys(browsers).some(type => {
   const ret = (new RegExp(type)).test(navigator.userAgent);
   ret && (browser = browsers[type]);
   return ret;
});

export { browser };
```

**Pull request merged**